### ISSUE TITLE
Common content for "Neural Networks for Solving Differential Equations in Classical Mechanics" and "Schrodinger Equation"

### ISSUE THREAD

**Issue by karanprime** [GDS-Education-Community-of-Practice/DSECOP#7]
Since @juliebutlerhartley and I are working on modules with similar ML techniques and applications, we will develop some content that will be used in both the modules. This will be divided as follows:
1) Content about solving diff eq: Analytical methods, boundary conditions (maybe even numerical methods) - by @juliebutlerhartley 
2) Content about ML models: Neural nets, loss functions, things like overfitting, physics informed models (loss functions with PDE constraints) etc. - by @karanprime

The plan is that these two points will serve as Lesson 1 and Lesson 2 in our modules. Subsequent lessons will be about the physics applications (classical mechanics and quantum mechanics).
